Airports/EGPA/Runway.txt

```diff
@@ -1,2 +1,2 @@
-09  27  089 269 N058.57.28.030 W002.54.36.440 N058.57.29.830 W002.53.19.510
-14  32  142 322 N058.57.37.820 W002.54.33.190 N058.57.23.920 W002.54.10.920
\ No newline at end of file
+09  27  087 267 N058.57.28.030 W002.54.36.440 N058.57.29.830 W002.53.19.510
+14  32  140 320 N058.57.37.810 W002.54.33.180 N058.57.23.920 W002.54.10.930
```

Comment:
```suggestion
09  27  089 269 N058.57.28.030 W002.54.36.440 N058.57.29.830 W002.53.19.510
14  32  142 322 N058.57.37.810 W002.54.33.180 N058.57.23.920 W002.54.10.930
```
The headings here remain the same as they're magnetic rather than true headings, which haven't changed this AIRAC.